src/cf.data.pre

```diff
@@ -4765,6 +4765,26 @@ DOC_START
 				not available. Consider encoding the logged
 				value because Issuer often has spaces.
 
+		ssl::<cert
+				The received server x509 certificate in PEM
+				format, including BEGIN and END lines. The
+				logged certificate may have failed validation
+				and may not be trusted by Squid. This field
+				does not include any intermediate certificates
+				that may have been received from the server or
+				fetched during certificate validation process.
+
+				Currently, Squid only collects server
+				certificates during step3 of SslBump
+				processing; connections that were not subject
+				to ssl_bump rules or that did not match a peek
+				or stare rule at step2 will not have the
+				server certificate information.
+
+				The default encoding for this field is
+				multi-line. When Squid cannot log a
```

Comment:
I think the default encoding should not be multi-line because we should not break access.log line-based format by _default_ and because a "quoted string" encoding is easy to support (by adding `quote = 1` to the relevant Format.cc code IIRC). I have proposed using quoted string encoding before, but you insisted on using multi-line default. Why do you insist on it?

Reply:
Hi Alex,

Sorry if I made it seem as though I insisted on multi-line as the default.  I simply observed that multi-line was the default the way I copied and pasted it.  Please feel free to make the default whichever way you see best.

Thanks for accepting this pull request.

Reply:
The default encoding text was removed from the documentation file cf.data.pre.